media-libs/libsoundtouch/libsoundtouch-2.1.0.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="A library to adjust the sound tempo, pitch and playback rate"
+DESCRIPTION="SoundTouch is an open-source audio processing library for \
+changing the tempo, pitch and playback rates of audio streams or audio files. \
+The library additionally supports estimating stable beats-per-minute rates for \
+audio tracks.
+
+- Tempo (time stretch): Changes the sound to play at faster or slower tempo \
+than originally without affecting the sound pitch.
+- Pitch (key) : Changes the sound pitch or key while keeping the original \
+tempo (speed).
+Playback Rate : Changes both tempo and pitch together as if a vinyl disc was \
+played at different RPM rate."
+HOMEPAGE="https://www.surina.net/soundtouch"
```

Comment:
Oops, you forgot the trailing slash.
```diff
-HOMEPAGE="https://www.surina.net/soundtouch"
+HOMEPAGE="https://www.surina.net/soundtouch/"
```
That's because `curl --head` (or `curl -I`) redirects to the same URI with a trailing slash added, so it is better to add it in the recipe.